src/components/reference-book/page.cjsx

```diff
@@ -81,7 +81,7 @@ module.exports = React.createClass
   render: ->
     {courseId} = @context.router.getCurrentParams()
     # read the id from props, or failing that the url
-    cnxId = @getCNXId()
+    cnxId = @getCnxId()
```

Comment:
Hm, I thought it looked to much like a constant with all the uppercase letters and the "Id" didn't have enough visual segregation from the CNX.  The method is only used internally though so it's naming isn't terribly important.  I can revert